versions.go

```diff
@@ -1,26 +1,66 @@
 /*
 MongoDB Versions
 
-The MongoDBVersion type provides support for interacting with MongoDB
+The LegacyMongoDBVersion type provides support for interacting with MongoDB
```

Comment:
`MongoDBVersion`

Reply:
sometimes the refactor tools lead me astray :')